molgenis-core-ui/src/main/resources/css/molgenis.css

```diff
@@ -39,13 +39,14 @@ html {
 
 body {
 	/* Margin bottom by footer height */
-	margin-bottom: 40px;
+    margin-bottom: 40px !important;
```

Comment:
'!important'  is the nuclear option , try to increase the css specificity instead